crates/router/src/core/admin.rs

```diff
@@ -50,18 +50,34 @@ pub fn create_merchant_publishable_key() -> String {
     )
 }
 
+pub async fn insert_merchant_configs(
+    db: &dyn StorageInterface,
+    merchant_id: &String,
+) -> RouterResult<()> {
+    db.insert_config(configs::ConfigNew {
+        key: format!("{}_requires_cvv", merchant_id),
+        config: "true".to_string(),
+    })
+    .await
+    .change_context(errors::ApiErrorResponse::InternalServerError)
+    .attach_printable("Error while setting requires_cvv config")?;
+
+    db.insert_config(configs::ConfigNew {
+        key: utils::get_merchant_fingerprint_secret_key(merchant_id),
+        config: utils::generate_id(consts::FINGERPRINT_SECRET_LENGTH, "fs"),
+    })
```

Comment:
Aren't the fields supposed to be interchanged: the `key` has the fingerprint secret ID, and the `config` has the actual fingerprint secret value?

Reply:
cc: @prajjwalkumar17